model-metadata/UMass-HMLR.yml

```diff
@@ -19,27 +24,23 @@ model_contributors: [
     "affiliation": "Department of Biostatistics and Epidemiology, University of Massachussetts Amherst",
     "email": "nick@umass.edu"
   },
-  {
-    "name": "Benjamin W. Rogers",
-    "affiliation": "Department of Biostatistics and Epidemiology, University of Massachussetts Amherst",
-    "email": "bwrogers@umass.edu"
-  },
   {
     "name": "Thomas Robacker",
     "affiliation": "Department of Mathematics and Statistics, University of Massachussetts Amherst",
     "email": "trobacker@umass.edu"
   },
   {
-    "name": "Maryclare Griffin",
-    "affiliation": "Department of Mathematics and Statistics, University of Massachussetts Amherst",
-    "email": "maryclaregri@umass.edu"
+    "name": "Benjamin W. Rogers",
+    "affiliation": "Department of Biostatistics and Epidemiology, University of Massachussetts Amherst",
+    "email": "bwrogers@umass.edu"
   },
 ]
 license: "CC-BY-4.0"
 team_funding: "This work has been supported by the National Institutes of General Medical Sciences (R35GM119582) and the US Centers for Disease Control and Prevention’s Center for Forecasting and Outbreak Analytics (NU38FT000008). The content is solely the responsibility of the authors and does not necessarily represent the official views of NIGMS, the National Institutes of Health, or CDC."
-methods: "A Bayesian hierarchical multinomial logistic regression model for nowcasting COVID variants. Regression coefficents are modeled hierarchically across variants and locations."
+methods: "A Bayesian hierarchical multinomial logistic regression model for nowcasting COVID variants. Regression coefficents are modeled hierarchically across variants and locations. Model is adapted from Abousamra, Figgins and Bedford (2024)."
```

Comment:
Could we say a few ways in which it is similar to/different from their model? E.g. didn't we change the way the hierarchy was used (didn't they do it across variants where we do it across locations?).

Reply:
Maybe also, in place of the citation below, we could add a journal name here, for a bit more of a "complete" citation.